streams/src/test/java/org/apache/kafka/streams/state/internals/TimestampedKeyValueStoreBuilderTest.java

```diff
@@ -46,14 +44,16 @@
     @Mock(type = MockType.NICE)
     private KeyValueBytesStoreSupplier supplier;
     @Mock(type = MockType.NICE)
-    private KeyValueStore<Bytes, byte[]> inner;
+    private RocksDBTimestampedStore inner;
```

Comment:
The new wrapping would break the test. Using `RocksDBTimestampedStore` avoid the wrapping as `TimestampedByteStore` interface is implemented.